style-guide.md

```diff
@@ -620,3 +620,45 @@ things = create_things()
 
 # res = triple_len(things)
 ```
+
+# Docstrings
+
+## Preferred delimiters
+
+You **should** enclose docstrings with `"""` delimiters.  That is, you **should
+not** enclose docstrings with `'''` delimiters.
```

Comment:
:+1:

Reply:
I found https://github.com/PyCQA/pydocstyle this morning which can enforce this for us in CI.